tests/components/bluetooth/test_diagnostics.py

```diff
@@ -3,6 +3,7 @@
 
 from bleak.backends.scanner import AdvertisementData, BLEDevice
 from bluetooth_adapters import DEFAULT_ADDRESS
+from habluetooth import HaScanner
```

Comment:
Because we now replace the class early, the diagnostics change a bit.

We have to subclass it instead of patching it now since its no longer patchable.